src/test/java/uk/gov/hmcts/darts/retention/service/impl/RetentionPostServiceImplTest.java

```diff
@@ -305,11 +303,6 @@ void ok_BeforeCurrentRetentionDate_Judge() {
         assertEquals(10, savedRetention.getCreatedBy().getId());
         assertEquals(RetentionPolicyEnum.MANUAL.getPolicyKey(), savedRetention.getRetentionPolicyType().getFixedPolicyKey());
         assertEquals(MANUAL_OVERRIDE, savedRetention.getConfidenceCategory());
-        verify(caseRepository).save(courtCaseEntityArgumentCaptor.capture());
-        CourtCaseEntity savedCourtCase = courtCaseEntityArgumentCaptor.getValue();
-        assertEquals(RetentionConfidenceReasonEnum.MANUAL_OVERRIDE, savedCourtCase.getRetConfReason());
-        assertEquals(RetentionConfidenceScoreEnum.CASE_PERFECTLY_CLOSED, savedCourtCase.getRetConfScore());
-        assertEquals(CURRENT_DATE_TIME, savedCourtCase.getRetConfUpdatedTs());
```

Comment:
Author note: The class under test no longer has responsibility for setting these attributes, therefore the assertions are removed here. Dedicated unit tests exist in `RetentionServiceImplTest` to verify these attributes are set correctly.

Similar comment applies to the other tests in this test class.